package.json

```diff
@@ -11,16 +11,12 @@
   },
   "peerDependencies": {
     "eslint": ">= 9",
-    "typescript": ">= 5"
-  },
-  "devDependencies": {
+    "typescript": ">= 5",
```

Comment:
@nick-atomix I'm otherwise happy with this, but I don't actually know why we have ts installed on an otherwise JS only package. I get we use the ts-eslint package, but that doesn't actually require ts.

Could be nice to remove.